reports/templates/header.php

```diff
@@ -68,7 +68,7 @@
 <script type="text/javascript" src="js/plugins/jquery.floatThead.min.js"></script>
 <script type="text/javascript" src="js/common.js"></script>
 <script type="text/javascript">
-Date.format = '<?php echo return_datepicker_date_format(); ?>';
+Date.format = '<?php // echo return_datepicker_date_format(); ?>';
```

Comment:
Why is this commented out?

Reply:
That piece of code is what's breaking the report view. I tried to figure out why this particular call to the function is breaking the page. It works in all the other modules. But since datapicker is not used in the reports module, this call is unnecessary. So we're commenting it out for now.